core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -393,6 +388,8 @@ public FileInfo getFileInfo(AlluxioURI path)
   }
 
   /**
+   * NOTE: {@link #mInodeTree} should already be locked before calling this method.
```

Comment:
I don't think we need this because we have `@GuardedBy("mInodeTree")` already. 
I deleted this line in previous PR.